### ISSUE TITLE
not fetching the actual tournament template inside the account

### ISSUE THREAD

**Issue by asish1346** [WubbaLubbaDubDubDev/notpixel_bot_advanced#37]
NotPixel | 2024-11-30 21:44:56 | INFO     | tapper.py: 1169 - Liza | Painting started
NotPixel | 2024-11-30 21:44:57 | SUCCESS  | tapper.py: 987 - Liza | Image downloaded and saved to: app_data/images/1467975528.png
NotPixel | 2024-11-30 21:45:00 | SUCCESS  | tapper.py: 1230 - Liza | Painted on (x=116, y=150) with color #FFD635| Reward: None 🟨
NotPixel | 2024-11-30 21:45:07 | INFO     | tapper.py: 1264 - Liza | Painting completed | Total repaints: 104

**Comment by WubbaLubbaDubDubDev** [comment]
> NotPixel | 2024-11-30 21:44:56 | INFO     | tapper.py: 1169 - Liza | Painting started
> NotPixel | 2024-11-30 21:44:57 | SUCCESS  | tapper.py: 987 - Liza | Image downloaded and saved to: app_data/images/1467975528.png
> NotPixel | 2024-11-30 21:45:00 | SUCCESS  | tapper.py: 1230 - Liza | Painted on (x=116, y=150) with color #FFD635| Reward: None 🟨
> NotPixel | 2024-11-30 21:45:07 | INFO     | tapper.py: 1264 - Liza | Painting completed | Total repaints: 104

I am currently resolving this issue.

**Comment by WubbaLubbaDubDubDev** [comment]
@asish1346 
Fixed the issue, but the reward will be Null for now, as there is currently no way to view the reward for coloring.

**Comment by asish1346** [comment]
After the painting the balance increases so add logging of balance after paint l, it may be convenient

**Comment by WubbaLubbaDubDubDev** [comment]
> After the painting the balance increases so add logging of balance after paint l, it may be convenient 

No, after coloring the tournament templates, the balance does not increase. Therefore, it is not possible to calculate the profit.
